io.github.Archeb.opentrace.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.Archeb.opentrace</id>
+  <name>OpenTrace</name>
+  <summary>OpenTrace is a cross-platform GUI wrapper for NextTrace. Bringing you a sense of familiarity.</summary>
```

Comment:
Summary shouldn't be this long and should not end with full stop https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines#no-weird-formatting-1